src/handlers/profile/mod.rs

```diff
@@ -13,8 +13,10 @@ pub mod reverse;
 pub mod suggestions;
 pub mod utils;
 
-/// List of allowed name zones
-pub const ALLOWED_ZONES: [&str; 1] = ["wcn.id"];
+/// List of allowed name zones for lookup
+pub const ALLOWED_ZONES: [&str; 2] = ["reown.id", "wcn.id"];
+/// Main zone for new names registration and suggestions
+pub const MAIN_ZONE: &str = ALLOWED_ZONES[0];
```

Comment:
Suggest this to improve explicitness of using reown.id as main zone:

```suggestion
const REOWN_ID: &str = "reown.id";
const WCN_ID: &str = "wcn.id";
/// List of allowed name zones for lookup
pub const ALLOWED_ZONES: [&str; 2] = [REOWN_ID, WCN_ID];
/// Main zone for new names registration and suggestions
pub const MAIN_ZONE: &str = REOWN_ID;
```

Reply:
Makes sense! Thanks for the comment.